### PR TITLE
Fix for blog posts api + added django-taggit-serializer

### LINKED ISSUE #502: [bug] Blog posts don't appear once created
1. Login the admin interface
2. Add a new blog post (make sure it's marked as visible)
3. Check to see if the new blog post was created

**Expected:**
The newly created blog post can be retrieved through the API

**Actual:**
The newly created blog post isn't returned by the API

### PR DESCRIPTION
**Pull request by mateesville93**
### What does it fix?

Closes #502

### CODE REVIEW

File: api/requirements.txt [hunk 1]
```diff
@@ -4,44 +4,103 @@
 #
 #    pip-compile --output-file=requirements.txt requirements.in
 #
-boto3==1.14.10            # via -r requirements.in
```
Comment (tudoramariei) [inline api/requirements.txt:7]: You'll have to run a `pip-compile` on the `requirements-dev` files as well.

**Comment by tudoramariei** [comment]
Please run a `make black`. We want to keep the formatting uniform across the codebase, even if that means having to pick sides in the `'` vs `"` war.

**Comment by tudoramariei** [comment]
After formatting the code with make black and updating the requirements-dev.txt file, the code should be good to go. :shipit: 
Regarding git, I'd also ask you to squash your commits into one to keep the git history clean. Also, look out for what account you're using since it looks like you have commits from two users now. 
Thanks a lot for this PR! If you have any question, let me know 😉

**Comment by mateesville93** [comment]
Thanks for info @tudoramariei I'll do it tomorrow.

**Review by tudoramariei — APPROVED** [review]

**Comment by tudoramariei** [comment]
Thanks again for the PR and the quick work 🚀  
Also, if you want to get more involved in our projects, we have an online HackDay tomorrow, from 10
If you want to join the community, there's a registration form for this 😄 
https://tfsg.code4.ro/ro/hackday/?where=online

**Pull request merged**